packages/firefox/firefox-bin.pacscript

```diff
@@ -0,0 +1,24 @@
+name="firefox-bin"
+version="92.0.1"
+maintainer="aranym <aranym1@pm.me>"
+url="https://download-installer.cdn.mozilla.net/pub/firefox/releases/${version}/linux-x86_64/en-US/firefox-${version}.tar.bz2"
+depends="lsb-release libatk1.0-0 libc6 libcairo-gobject2 libcairo2 libdbus-1-3 libdbus-glib-1-2 libfontconfig1 libfreetype6 libgcc-s1 libgdk-pixbuf-2.0-0 libglib2.0-0 libgtk-3-0 libharfbuzz0b libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb-shm0 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrender1"
+description="Safe and easy web browser from Mozilla"
+hash="a3ba62122a6cde1d7d2ebf96e765ea57ea3e9a312d2161e0ac70923946844689"
+
+prepare() {
+    sudo mkdir -p /usr/src/pacstall/firefox-bin/usr/share/pixmaps
+    sudo mkdir -p /usr/src/pacstall/firefox-bin/usr/bin
+    sudo mkdir -p /usr/src/pacstall/firefox-bin/usr/share/applications
+    wget -q https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/firefox/firefox-bin.desktop
+}
+
+build() {
+    true
+}
+
+install() {
+    sudo mv ./browser/chrome/icons/default/default128.png /usr/src/pacstall/firefox-bin/usr/share/pixmaps
+    sudo mv ./firefox-bin.desktop /usr/src/pacstall/firefox-bin/usr/share/applications
+    sudo mv ./* /usr/src/pacstall/firefox-bin/usr/bin
+}
```

Comment:
Instead of doing `mkdir` and `mv`, just use the `install` command, it'll do both at once.

Example:
```bash
sudo install -D foo -t bar/
```
This will move the `foo` to the directory called `bar/` which isn't made yet.